README.md

```diff
@@ -0,0 +1 @@
+# collaborativeProject
```

Comment:
The link to the said README template is missing. Please make sure to add the link to the template so that the student is able to access it. Please also note that the project is said to be invalid if a README file is missing.